coderd/workspacebuilds.go

```diff
@@ -635,6 +635,7 @@ func (api *API) workspaceBuildState(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// TODO: why?
```

Comment:
Drive-by: I don't understand why this endpoint has to authorize a template?
Looks like copy-pasta.

Reply:
It's *not*. If you can get/push the workspace build state, you can potentially do nasty things. See: https://github.com/coder/security/issues/2

Reply:
Oooh, TIL!